### ISSUE TITLE
Print Barcode

### ISSUE THREAD

**Issue by HWSTS** [mazenrashed/Printooth#45]
Salam mazin,

is this lib support barcode printing ?

Thanks in advance

**Comment by RaRoPe** [comment]
Yes, @appsdev85, this lib support printing images in general, barcode is one of them =D
